src/App.js

```diff
@@ -4,27 +4,61 @@ import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 
 
+
 function App() {
   const [fighters, setFighters] = useState([]);
  
-  React.useEffect(() => {
+      useEffect(() => {
     axios.get('https://api.octagon-api.com/fighters').then((response) => {
       setFighters(response.data);
     });
 
   }, []);
-  {Object.values(data).map(fighter => fighter.name)};
-  console.log(fighters);
-  <ul>
-  
-   
-  <li key={fighters.name}>{fighters.name}</li>
-
-  </ul>
-  
-  console.log(fighters);
 
+return(
+    <div>
+     
+     <table
+      style={{
+        borderCollapse: "collapse",
+        border: "2px solid rgb(140, 140, 140)",
+        fontFamily: "sans-serif",
+        fontSize: "0.8rem",
+        letterSpacing: "1px",
+      }}
+    >
+      <th><h1>Fighters list</h1></th>
+      <tbody>
+  {Object.values(fighters).map((fighter, id) => {
+    return (
+      <div style={{display: "inline-flex",
+        margin: 0,
```

Comment:
Because there are so many styles inline it might worth consider to move them into a separet file.

Ex of what it would look like,

import styles from '.styles.module.css';

```
<div style={styles.container>
    <tr key={id} style={styles.row}>
    ...
    </tr>
  </div>

```